libraries/standard/mqtt/include/types/iot_mqtt_types.h

```diff
@@ -685,17 +685,135 @@ typedef struct IotMqttConnectInfo
     uint16_t passwordLength; /**< @brief Length of #IotMqttConnectInfo_t.pPassword. */
 } IotMqttConnectInfo_t;
 
-#if IOT_MQTT_ENABLE_SERIALIZER_OVERRIDES == 1
-
 /**
  * @cond DOXYGEN_IGNORE
  * Doxygen should ignore this section.
  *
  * Forward declaration of the internal MQTT packet structure.
  */
-    struct _mqttPacket;
+struct _mqttPacket;
 /** @endcond */
 
+/**
+ * @brief Get the MQTT packet type from a stream of bytes off the network.
+ *
+ * @param[in] pNetworkConnection Reference to the network connection.
+ * @param[in] pNetworkInterface Function pointers used to interact with the
+ * network.
+ */
+typedef uint8_t ( * IotMqttGetPacketTypeFunc_t )(
```

Comment:
Nit: naming of these function pointers. Some have "Func" at the end and some don't.

I'd prefer not having the "Func" at the end, so just `IotMqttGetPacketType_t` and `IotMqttGetRemainingLength_t` etc,

Reply:
Good point, I agree the `Func` is redundant and will remove.